packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -89,36 +93,99 @@ describe('App', () => {
     })
 
     test('Active Data component is not rendered', async () => {
-        constants.ACTIVE_DATA_ENABLED = false
-        useMultiSite.mockImplementation(() => resultUseMultiSite)
-        renderWithProviders(
-            <App targetLocale={DEFAULT_LOCALE} defaultLocale={DEFAULT_LOCALE} messages={messages}>
-                <p>Any children here</p>
-            </App>
-        )
-        await waitFor(() =>
-            expect(document.getElementById('headActiveData')).not.toBeInTheDocument()
-        )
-        await waitFor(() => expect(document.getElementById('dwanalytics')).not.toBeInTheDocument())
-        await waitFor(() => expect(document.getElementById('dwac')).not.toBeInTheDocument())
-        expect(screen.getByText('Any children here')).toBeInTheDocument()
+        jest.doMock('@salesforce/retail-react-app/app/constants', () => {
```

Comment:
This refactor addressed the eslint error
![image](https://github.com/user-attachments/assets/8cec6952-2488-4a68-b24e-bddb60c10f3a)

Reply:
Using doMock will allow us to mock the value of a constant without modifying a constant value directly